teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Docking software (no plural)</li><li>I think you need to split this cell in two so Contents in Practical gets read in the table of contents. This also applies to other cells! Every header of subheader must start its own cell.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>